validate_1.0.0-rc.2/checksum.py

```diff
@@ -0,0 +1,50 @@
+# !/usr/bin/env python3
+# -*- coding: utf-8 -*-
+ 
+ 
+import hashlib
+import os
+ 
+def get_checksum(filename, hash_function):
+    """Generate checksum for file baed on hash function (MD5 or SHA256).
+ 
+    Args:
+        filename (str): Path to file that will have the checksum generated.
+        hash_function (str):  Hash function name - supports MD5 or SHA256
+ 
+    Returns:
+        str`: Checksum based on Hash function of choice.
+ 
+    Raises:
+        Exception: Invalid hash function is entered.
+ 
+    """
+    hash_function = hash_function.lower()
+ 
+    with open(filename, "rb") as f:
+        bytes = f.read()  # read file as bytes
+        if hash_function == "sha256":
+            readable_hash = hashlib.sha256(bytes).hexdigest()
+        else:
+            Raise("{} is an invalid hash function. Please Enter MD5 or SHA256")
+ 
+    return readable_hash
+
+hashcode = "checksum/hash.tar.gz"
+hashtest = "462000e7b041f54b1b8e4ef59e4372f62f9bb2d293c3f9122a4f19c46edadf17"
```

Comment:
check my comment below, don't think this checksum would match considering your tar command in the readme

Reply:
checksum now succeeds